plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/metastore/TestDeltaLakeMetastoreAccessOperations.java

```diff
@@ -90,6 +92,7 @@ public void testSelectWithFilter()
         assertMetastoreInvocations("SELECT * FROM test_select_from_where WHERE age = 2",
```

Comment:
We need to test several `SELECT` operations in a row on the same table to ensure that the `REPLACE_TABLE` is called only the first time when we select from the table.